stylesheets/_component.forms.scss

```diff
@@ -328,13 +330,15 @@ fieldset {
 // based on the remaining 9 column grid
 
     // default width
-    .control__label + .controls > .form__control:not(.checkbox) {
```

Comment:
The `:not(.checkbox)` is needed here to fix checkbox widths and bg colours in FF and IE.